Tests/kaas/k8s-version-policy/k8s_version_policy.py

```diff
@@ -171,7 +171,7 @@ def __str__(self):
 
 
 def parse_version(version_str: str) -> K8sVersion:
-    cleansed = version_str.strip().removeprefix("v")
+    cleansed = version_str.strip().removeprefix("v").split("+")[0]
```

Comment:
Thanks for spotting this!

```suggestion
    cleansed = version_str.strip().removeprefix("v").split("+")[0]  # remove leading v as well as build info
```

Reply:
It still won't recognize `-prerelease`, but maybe pre-releases are not relevant here.

Reply:
not really into the scs policies, but from what I read it mostly targets operative environments. I would not expect any prerelease versions in such environments.

Reply:
I would also expect (even if it is not explicitly defined) to not require any prerelease versions as valid versions. As far as I understand, the goal is to have timely updates and not lag behind but stay on "upstream-supported" releases. using pre-releases will reduce stability and supportability.

Reply:
Yes, I agree!